src/devices/custom_devices_diy.ts

```diff
@@ -1175,6 +1175,17 @@ const definitions: DefinitionWithExtend[] = [
             }),
         ],
     },
+    {
+        zigbeeModel: ['FanBee1', 'Fanbox2'],
+        model: 'FanBee',
+        vendor: 'Lorenz Brun',
+        description: 'Fan with valve',
```

Comment:
Can you provide a link to this device? Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt/pull/24483

Reply:
This is a custom device built by me. I'll eventually open-source the design files and firmware, but right now there is no public docs or anything for it.

Reply:
For those type of devices, you can use the external converters (there is no point of adding it to the list of officially supported devices if nobody can get it)

Reply:
I know, but for the tests we need a device which exercises this behavior. There is an Inovelli fan which looks like it could also use this fan control model but as I do not own one of them I didn't feel comfortable changing it over to the new model (also this is a breaking change for anyone using the MQTT API).

I do intend on making this device publicly available.

Reply:
> I do intend on making this device publicly available.

Good, then I will merge it once this is done.

Reply:
https://github.com/lorenz/fanbee